quick-start.md

```diff
@@ -2,222 +2,87 @@
 
-After the installation is complete, you can test the installation by checking the Kontena CLI version with `kontena version`.
+<img src="_images/kontena-cloud-create-org.png" alt="Create Organization" />
 
-## Step 2. Create a Kontena Platform
+## Step 3. Create a Kontena Platform
 
-The easiest (and preferred) way to provision Kontena Platform Master is to use the built-in Kontena Cloud Platform provision feature of Kontena CLI. In this guide, we will provision Kontena Platform to Kontena Cloud and nodes to the local development environment using [Vagrant](https://www.vagrantup.com/). If you want to install nodes to some other environment, please see [Installing Kontena](using-kontena/install-nodes/) documentation.
+Kontena Platform will abstract all available compute resources and make these resources available to your container workloads. To create your first Kontena Platform, just open the 'Create Platform' dialog and fill in the platform name. For testing purposes `Mini` platform type is enough.
 
-Since we will be using Vagrant, please ensure you have Vagrant 1.6 or later installed. For more details, see the official [Vagrant installation docs](https://docs.vagrantup.com/v2/installation/index.html).
+<img src="_images/kontena-cloud-create-platform.png" alt="Create Platform" />
 
-After Vagrant is installed, you can install required plugins to Kontena CLI.
+After clicking 'Create' button Kontena Platform will be shortly provisioned to Kontena Cloud.
 
-```
-$ kontena plugin install cloud
-$ kontena plugin install vagrant
-```
+## Step 4. Install Kontena Nodes
 
-Before you can create a Kontena Cloud Platform you need to login:
+When the platform is ready you'll need some Kontena Nodes to run your containerized workloads.
 
-```
-$ kontena cloud login
-```
+After selecting Kontena Cloud as provider and clicking 'Add nodes' button, terminal windows should open and you can choose a node type and number of nodes you want to create.
 
-After login is completed succesfully you can start Kontena Platform provisioning with following command:
+<img src="_images/kontena-cloud-provision-nodes.png" alt="Provision Nodes" />
 
-```
-$ kontena cloud platform create quick-start
-> This will create managed platform to Kontena Cloud, proceed? Yes
-> Choose organization: my-username (you)
-> Choose region: EU West
-> Initial platform size (number of nodes): 1 (dev/test)
- [done] Creating platform quick-start to region eu-west-1
- [done] Waiting for platform quick-start to come online
- [done] Switching to use platform my-username/quick-start
-```
-
-During the installation process you will have the option to select region where Kontena Platform is provisioned. It's recommended to select closest region for you to minimize latency between your local development environment and Kontena Platform.
```

Comment:
During the installation process you will have the option to select the region where your Kontena Platform is provisioned. It's recommended to select the closest region for you to minimize latency between your local development environment and your Kontena Platform.

Reply:
@saana-kontenacm texts with red background are to be removed ;)

Reply:
ok, thanks. I realised that part way through :)